pkg/tests/end_to_end_tests/create_test.go

```diff
@@ -15,7 +15,8 @@ import (
 	pgx "github.com/jackc/pgx/v4"
 	"github.com/jackc/pgx/v4/pgxpool"
 	_ "github.com/jackc/pgx/v4/stdlib"
-	. "github.com/timescale/promscale/pkg/pgmodel"
+	igstr "github.com/timescale/promscale/pkg/pgmodel/ingestor"
```

Comment:
Again, I prefer to change variable names if they are clashing to package names. 
Also, we are using `ingr`, `ingstr`,  and here `igstr` for a short version of the word `ingestor`. This needs to be consistent across the project.